src/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -72,6 +72,11 @@ class DataObjectController extends ElementControllerBase implements KernelContro
 
     private array $classFieldDefinitions = [];
 
+    public function __construct(
+        protected ElementServiceInterface $elementService
+    ) {
+    }
```

Comment:
could all these be moved under `ElementControllerBase`?

Reply:
yes, good point. moved it 😊